DESCRIPTION.md

```diff
@@ -8,6 +8,10 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
+- v2.8.4(unreleased)
+  - Fixed a bug where the permission of the file downloaded via GET command is changed
+  - Reworked authentication internals to allow users to plug custom key-pair authenticators
```

Comment:
should we bump to 2.9.0 as we're introducing a new feature?

Reply:
We talked through this offline